packages/networks/src/networks/pulsechain.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const pulsechain: NetworkConfig = {
+    chain: 'pulsechain',
+    description: 'pulsechain mainnet.',
+    explorer: {
+        name: 'Pulsechain',
+        urls: {
+            address: (address: string) =>
+                `https://scan.pulsechain.com/address/${address}`,
+            base: `https://scan.pulsechain.com/`,
+            token: (address: string, holder: string) =>
+                `https://scan.pulsechain.com/token/${address}?a=${holder}`,
+            transaction: (hash: string) => `https://scan.pulsechain.com/tx/${hash}`,
+        },
+    },
+    featured: true,
+    fullySubsidizedGas: false,
+    id: 369,
+    isTestNetwork: false,
+    keyManagerAddress: '',
+    maxFreeClaimCost: 10000,
+    multisig: '',
+    name: 'pulsechain',
+    nativeCurrency: {
+        coingecko: 'pulsechain',
+        decimals: 18,
+        name: 'PLS',
+        symbol: 'PLS',
+    },
+    previousDeploys: [],
+    provider: 'https://rpc.unlock-protocol.com/369',
+    publicLockVersionToDeploy: 13,
+    publicProvider: 'https://scan.pulsechain.com',
+    startBlock: 2247300,
+    // Graph can be found at https://graph.pulsechain.com/

```

Comment:
Since this is not a standard https://thegraph.com/ setup you need to send us docs on how to set it up.

Reply:
I will try and get some more info for you on this side of things. Is this a nice to have or required?

Reply:
Required for the frontend to work

Reply:
Ok, so there is a community graph node that can be used. I have spoken to the team running this and they have advised while there is no end-user UI at present if you are able to provide the subgraph code you want hosted they will run this up.

Reply:
Sorry this is too complex for us to maintain so until there is a proper graph deployment we won't merge this PR.